engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1859,14 +1932,14 @@ static void LogFrameBufferError(GLenum status)
         OpenGLContext* context = (OpenGLContext*) _context;
 
         if (glBindVertexArray)
-            glBindVertexArray(context->m_GlobalVAO);
+            glBindVertexArray(GetGLHandle(context, context->m_GlobalVAO));
 
-        if (!(context->m_ModificationVersion == vertex_declaration->m_ModificationVersion && vertex_declaration->m_BoundForProgram == ((OpenGLProgram*) program)->m_Id))
+        if (context->m_ModificationVersion != vertex_declaration->m_ModificationVersion || vertex_declaration->m_BoundForProgram != program)
```

Comment:
Fixed condition and simplified it.
IMO error was in the part `vertex_declaration->m_BoundForProgram == ((OpenGLProgram*) program)->m_Id)`. `m_BoundForProgram` stores `HProgram`.